ui/src/assets/i18n/en.json

```diff
@@ -653,7 +653,7 @@
   "timeline_filter_project": "Mute events on: ",
   "timeline_filter_updated": "Filter updated",
   "timeline_filter_project_select": "Select a project",
-  "toast_http_error_request_id": "Request ID",
+  "toast_http_error_request_id": "with this request ID:",
```

Comment:
I think we can let the old traduction for the case there is no message in the api config.